ios/Framework/Core/Asset/ListScreenlet/AssetListScreenlet.swift

```diff
@@ -13,51 +13,65 @@
  */
 import UIKit
 
-
+/// The AssetListScreenletDelegate protocol defines some methods that you use to manage the
+/// AssetListScreenlet events. All of them are optional.
 @objc public protocol AssetListScreenletDelegate : BaseScreenletDelegate {
 
 	/// Called when a page of assets is received. Note that this method may be called 
 	/// more than once; one call for each page received.
 	///
 	/// - Parameters:
-	///   - screenlet
-	///   - assets: list of assets.
+	///   - screenlet: Asset list screenlet instance.
+	///   - assets: List of assets.
```

Comment:
Improves readability when developers are viewing documentation in Xcode :)
<img width="639" alt="quick documentation in xcode" src="https://cloud.githubusercontent.com/assets/15845466/24273830/e55e06ca-1024-11e7-8d7b-12e0be0cce67.png">